File: GGAndroid_2.0/app/src/main/java/io/ganguo/incubator/view/activity/ApkInfoActivity.java
package io.ganguo.incubator.view.activity;

import android.content.Context;
import android.content.Intent;

import io.ganguo.incubator.viewmodel.activity.ApkInfoViewModel;
import io.ganguo.viewmodel.databinding.IncludeHfRecyclerBinding;
import io.ganguo.vmodel.ViewModelActivity;

/**
 * Apk相关数据信息
 * Created by leo on 2018/11/1.
 */
public class ApkInfoActivity extends ViewModelActivity<IncludeHfRecyclerBinding, ApkInfoViewModel> {

    /**
     * function:create intent
     *
     * @param context
     */
    public static Intent createIntent(Context context) {
        Intent intent = new Intent(context, ApkInfoActivity.class);
        return intent;
    }


    @Override
    public ApkInfoViewModel createViewModel() {
        return new ApkInfoViewModel();
    }

    @Override
    public void onViewAttached(ApkInfoViewModel viewModel) {

    }
}

File: GGAndroid_3.0/main/app/src/main/java/io/ganguo/incubator/view/general/activity/ApkInfoActivity.kt
package io.ganguo.incubator.view.general.activity

import io.ganguo.incubator.viewmodel.general.activity.ApkInfoViewModel
import io.ganguo.viewmodel.core.BaseViewModel
import io.ganguo.viewmodel.pack.base.activity.BaseVModelActivity
import io.ganguo.viewmodel.databinding.IncludeHfRecyclerBinding

/**
 * <pre>
 * author : leo
 * time   : 2018/11/1
 * desc   : Apk及手机相关参数展示
 * </pre>
 */
class ApkInfoActivity : BaseVModelActivity<IncludeHfRecyclerBinding, ApkInfoViewModel>() {
    override fun createViewModel(): ApkInfoViewModel {
        return ApkInfoViewModel()
    }

    override fun onViewAttached(viewModel: BaseViewModel<Any>) {
    }

}
